### ISSUE TITLE
jax's obnoxious list of bugs

### ISSUE THREAD

**Issue by jaxatto** [walaura/watermelon-pizza#33]
tbh i didn't want to spam you with 20 tickets so

a11y
- default browser [focus state](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html) does not meet [contrast requirements (3/1)](https://www.w3.org/WAI/WCAG21/Techniques/css/C40), you can do some pretty things with this!
- content under `<h1>` should be inside of `<p>`
- don't nest a `<footer>` inside of a `<footer>`, there can only be one per page ONLY ONE
- [title tag missing from links](https://www.w3.org/TR/WCAG20-TECHS/H33.html)
- not a bug but suggestion, `<s>` won't read to screen readers as marked out, alternatives recommend CSS shenanigans to hide it from screen readers, or otherwise note in a tittle maybe?
- [warn users that a new window will open when selecting a link](https://www.w3.org/TR/WCAG20-TECHS/G201.html)
- [add reduced animation query](https://www.w3.org/WAI/WCAG21/Techniques/css/C39), very helpful for some users

"artistry"
- squiggles on the little links at the bottom don't touch
- originally i found this but i think you fixed it
<img width="316" alt="Screen Shot 2019-12-15 at 6 47 51 PM" src="https://user-images.githubusercontent.com/6673768/70912811-5c4b6d00-1fda-11ea-9ae6-88601b49cd4c.png">

other
- why is your green variable called pink tho
- should you use `&copy;` instead of the direct symbol itself?

ok thank you for listening to my shenanigans, hope this helps

**Comment by walaura** [comment]
hey thanks for reporting this!!! will get to them

PS: footer in a footer makes PERFECT SENSE and the web platform is being A JERK TO ME here but fine ill fix it
